app/src/main/java/com/android/voyageur/ui/trip/Tabs.kt

```diff
@@ -77,30 +78,49 @@ fun TopTabs(
         modifier = Modifier.fillMaxWidth().testTag("tabRow"),
     ) {
       // Create each tab with a Tab composable
-      tabs.forEachIndexed { index, title ->
-        Tab(
-            selected = navigationActions.getNavigationState().currentTabIndexForTrip == index,
-            onClick = { navigationActions.getNavigationState().currentTabIndexForTrip = index },
-            text = { Text(title) })
+      if (navigationActions.getNavigationState().isReadOnlyView) {
+        readOnlyTabs.forEachIndexed { index, title ->
+          Tab(
+              selected = navigationActions.getNavigationState().currentTabIndexForTrip == index,
+              onClick = { navigationActions.getNavigationState().currentTabIndexForTrip = index },
+              text = { Text(title) })
+        }
+      } else {
+        tabs.forEachIndexed { index, title ->
+          Tab(
+              selected = navigationActions.getNavigationState().currentTabIndexForTrip == index,
+              onClick = { navigationActions.getNavigationState().currentTabIndexForTrip = index },
+              text = { Text(title) })
+        }
```

Comment:
let's avoid duplicated code by doing something like 
`val tabsToDisplay = if (...) readOnlyTabs else tabs`
`tabsToDisplay.forEachIndexed...`

Reply:
You're right, it's better this way. I changed it.